client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -188,37 +198,28 @@ function CheckoutSummaryFeaturesList( props: {
 			) }
 			<CheckoutSummaryFeaturesListItem>
 				<WPCheckoutCheckIcon id="features-list-support-text" />
-				<SupportText hasPlanInCart={ hasPlanInCart } isJetpackNotAtomic={ isJetpackNotAtomic } />
+				{ translate( 'Customer support via email' ) }
 			</CheckoutSummaryFeaturesListItem>
 
 			{ ! hasPlanInCart && <CheckoutSummaryChatIfAvailable siteId={ siteId } /> }
 
-			{ showRefundText && (
-				<CheckoutSummaryFeaturesListItem>
-					<WPCheckoutCheckIcon id="features-list-refund-text" />
-					{ refundText }
-				</CheckoutSummaryFeaturesListItem>
-			) }
+			{ uniqueRefundableProducts.length &&
+				uniqueRefundableProducts.map( ( product ) => {
+					const productName = isDomainProduct( product )
+						? translate( 'Domain Registration', { textOnly: true } )
+						: product.product_name;
```

Comment:
Why not use the domain name in `product.meta` to make it clearer what the refund window is in each case? I feel that would be more useful, especially for cases where users may not be 100% clear on what a domain registration is.

Reply:
As it currently stands, if the customer has more than one domain in their cart, we still just display one row in the checkout summary.

If we want to revert to the previous logic however, then I agree that this is probably a better idea.

Any thoughts on collapsing multiple domains into a single row vs showing on row per domain?

Reply:
My thought was driven by the relevance of the wording to an end user for two reasons:
1. I don't know how often users actually end up buying multiple domains -- most of our users only have one domain total, so are very unlikely to purchase two at the same time
2. When users are buying a domain-related product, it may be more relevant to refer to the domain name itself rather than our internal name for the product, as that may make more immediate sense to the user

Maybe @Automattic/nomado can weigh in with their thoughts on which approach would be better.

Reply:
Ended up rolling back the grouping logic. I think users can tolerate some repetition as long as the relevance to them is clear. By printing the actual domain on each row (instead of `Domain Registration`), I think we can consider the information relevant rather than just repetitive.

See my latest comment for current status https://github.com/Automattic/wp-calypso/pull/63489#issuecomment-1131869773